### PR TITLE
Added module descriptions and other minor changes to remove warning messages

### PR DESCRIPTION
**Pull request by foulox**
Please take a look at the changes I made

### CODE REVIEW

**Review by Saint-Fennec — APPROVED** [review]
Everything looks good.

File: src/test_scraper.py [hunk 1]
```diff
@@ -1,22 +1,39 @@
...
-        concert_list.append({'name': name, 'date': date, 'location': location})
-    return concert_list
+        fetched_concerts.append({'name': name, 'date': date, 'location': location})
+    return fetched_concerts
 
 # Example URL, replace with the actual Bowery Presents schedule page
-concerts_url = 'https://example.com/concerts'
-concert_list = scrape_concerts(concerts_url)
+CONCERTS_URL = 'https://example.com/concerts'
+concert_list = scrape_concerts(CONCERTS_URL)
```
Comment (foulox) [inline src/test_scraper.py:37]: Now we are reading the concerts calendar

File: src/test_scraper.py [hunk 2]
```diff
@@ -1,22 +1,39 @@
...
-        # Extract necessary information, e.g., band name, date, location. This depends on the website's HTML structure.
+        # Extract necessary information, e.g., band name, date,
+        # location. This depends on the website's HTML structure.
         name = concert.find('h2').text
         date = concert.find('span', class_='date').text
         location = concert.find('span', class_='location').text
-        concert_list.append({'name': name, 'date': date, 'location': location})
-    return concert_list
+        fetched_concerts.append({'name': name, 'date': date, 'location': location})
+    return fetched_concerts
```
Comment (foulox) [inline src/test_scraper.py:33]: I believe this is grabbing all the concerts

Comment (foulox) [inline src/test_scraper.py:33]: This is grabbing the name of each concert. Would you try to grab other important information?

**Pull request merged**